src/reports/biospecimen-request/biospecimenRequestStats.ts

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-console */
+import { Request, Response } from 'express';
+
+import EsInstance from '../../ElasticSearchClientInstance';
+import { reportGenerationErrorHandler } from '../../utils/errors';
+import getAvailableBiospecimensFromSqon from '../utils/getAvailableBiospecimensFromSqon';
+
+interface IBiospecimenDataByStudy {
+    study_code: string;
+    study_name: string;
+    nb_participants: number;
+    nb_available_samples: number;
+    nb_containers: number;
+}
+
+const biospecimenRequestStats = () => async (req: Request, res: Response): Promise<void> => {
+    console.time('biospecimenRequestStats');
+
+    const { sqon, projectId } = req.body;
+    const userId = req['kauth']?.grant?.access_token?.content?.sub;
+    const accessToken = req.headers.authorization;
+
+    const wantedFields = ['sample_id', 'study.study_code', 'study.study_name', 'participant_fhir_id', 'container_id'];
+
+    let esClient = null;
+
+    try {
+        esClient = EsInstance.getInstance();
```

Comment:
You should declare it outside of the try block. 

one of the reason is that you use it there in the catch block
```
catch (err) {
        reportGenerationErrorHandler(err, esClient);
    }
```